test-lib/test.js

```diff
@@ -1,9 +1,16 @@
 const fs = require('fs-extra');
 const path = require('path');
 
-fs.removeSync(path.join(__dirname, '/../test/node_modules'));
-fs.mkdirSync(path.join(__dirname, '/../test/node_modules'));
-fs.symlinkSync(path.join(__dirname, '/..'), path.join(__dirname, '/../test/node_modules/apostrophe'), 'dir');
+const testNodeModules = path.join(__dirname, '/../test/node_modules');
```

Comment:
We are now able to ship tests of what would happen if certain code existed in `node_modules` during unit tests, necessary to adequately test this scenario. We do it by putting it in `test/extra_node_modules`. That gets brought into the `test/node_modules` folder along with the symlink to apostrophe itself, which was already being built at test startup time.